inc/types/Matrix4.h

```diff
@@ -27,6 +27,9 @@ DEALINGS IN THE SOFTWARE.
 
 #include "CodalConfig.h"
 
+namespace codal
+{
```

Comment:
Technically there is a small risk here, as any file including `Matrix4.h` would need to ensure it is using the codal namespace as well.
However, had a seach in GitHub and this class was only used in an old implementation of the magnetometer/compass calibration, so none of the normal targets are using it (only really old, non updated, code).